supportedBackends/google/pipelines/common/src/test/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributesSpec.scala

```diff
@@ -315,56 +279,6 @@ class PipelinesApiConfigurationAttributesSpec extends AnyFlatSpec with CromwellT
     errorsList should contain("Virtual Private Cloud configuration is invalid. Missing keys: `network-label-key`.")
   }
 
-  it should "not parse memory-retry without error-keys" in {
-    val config =
-      ConfigFactory.parseString(
-        """
-          |memory-retry {
-          |   multiplier = 1.1
-          |}
-        """.stripMargin)
-
-    val exception = intercept[IllegalArgumentException with MessageAggregation] {
-      PipelinesApiConfigurationAttributes(googleConfig, config, "papi")
-    }
-    val errorsList = exception.errorMessages.toList
-    errorsList should contain("memory-retry configuration is invalid. No error-keys provided.")
-  }
-
-  it should "not allow a positive multiplier less than 1.0" in {
-    val config =
-      ConfigFactory.parseString(
-        """
-          |memory-retry {
-          |   error-keys = ["OutOfMemory", "Killed", "Exit123"]
-          |   multiplier = 0.5
-          |}
-        """.stripMargin)
-
-    val exception = intercept[IllegalArgumentException with MessageAggregation] {
-      PipelinesApiConfigurationAttributes(googleConfig, config, "papi")
-    }
-    val errorsList = exception.errorMessages.toList
-    errorsList should contain("Value 0.5 for memory-retry.multiplier should be greater than 1.0.")
-  }
-
-  it should "not allow multiplier negative multiplier" in {
-    val config =
-      ConfigFactory.parseString(
-        """
-          |memory-retry {
-          |   error-keys = ["OutOfMemory", "Killed", "Exit123"]
```

Comment:
OOI: where is this validation now?

Reply:
Does moving this out of the `PAPI` backend config and into general purpose config give the wrong impression and make it appear that this should now work on any backend?

Reply:
Yeah it can possibly give the wrong impression. My thought was that since in the future we would be running workflows on other backends, it would be good to make it generalized. I am still in two minds about this on whether to keep it in Papiv2 backend config or global config. What are your thoughts?

Reply:
If we start with global we can later add a per-backend override if we need it.

If we start with per-backend, we can't easily go the other way.